src/custom/components/Tokens/TokensTable.tsx

```diff
@@ -86,6 +96,17 @@ export default function TokenTable({
     [history]
   )
 
+  const { ErrorModal } = useErrorModal()
+
+  const { TransactionConfirmationModal, openModal, closeModal } = useTransactionConfirmationModal(
+    OperationType.APPROVE_TOKEN
+  )
+
+  const openTransactionConfirmationModal = useCallback(
```

Comment:
now sure this is needed since ur not doing anything/the only dep is the fn itself

Reply:
Good point :+1: